### ISSUE TITLE
AutoMix tutorial for ImageNet

### ISSUE THREAD

**Issue by 774911840** [Westlake-AI/openmixup#21]
Hi,i run AutoMix use Cifar100 dateset,but the image is low pixel.So i want to use ImageNet,i would appreciate it if you provide some help!

**Comment by Lupin1998** [comment]
Hi @774911840, thanks for your attention and appreciation of our [AutoMix](https://arxiv.org/abs/2103.13027). We have provided training [configs](https://github.com/Westlake-AI/openmixup/tree/main/configs/classification/imagenet/automix/basic) of AutoMix on ImageNet, and you can train ResNet variants by `CUDA_VISIBLE_DEVICES=0,1,2,3 bash tools/dist_train.sh ${CONFIG_FILE} 4`.

For example, AutoMix with ResNet-18: `CUDA_VISIBLE_DEVICES=0,1,2,3 bash tools/dist_train.sh configs/classification/imagenet/automix/basic/r18_l2_a2_near_lam_cat_mb_mlr1e_3_bb_mlr0.py 4`. Visualization of mixed samples will be saved in the relevant work_dirs. You can modify `max_epochs` for longer training time. Feel to ask me if you have more questions.

**Comment by Lupin1998** [comment]
Currently, we are busy working on other projections, thus, it might take a bit long time for us to collect pre-trained models, logs, and visualization results on ImageNet. You can refer to mixed samples of [AutoMix](https://arxiv.org/abs/2103.13027) provided on [Places205](https://github.com/Westlake-AI/openmixup/releases/tag/mixup-place205-weights), [iNat2017](https://github.com/Westlake-AI/openmixup/releases/tag/mixup-inat2017-weights), and [iNat2018](https://github.com/Westlake-AI/openmixup/releases/tag/mixup-inat2018-weights) @774911840. Please watch us for the latest update of `OpenMixup`.

**Comment by 774911840** [comment]
Thanks for answer,i know your means.Actually i am confused of `openmixup/data` structure for imagenet.
I paster dataset config `configs/classification/_base_/datasets/imagenet/basic_sz224_4xbs64.py` down here.
![image](https://user-images.githubusercontent.com/42885976/190589105-19cc30a5-56a4-445b-91ac-03885927ede1.png)

**Comment by Jacky1128** [comment]
Hi @774911840, I have viewed your questions above and here are my replies: 
1. For the training set of ImageNet, the folders for different categories need to be unzipped and be placed under `data/ImageNet/train`.
2. For the validation one, all the images should be placed under `data/ImageNet/val`.
3. The corresponding meta files ought to be put under `data/meta`.

Notice that the detailed guidelines for data preparation are given in [tutorial](https://openmixup.readthedocs.io/en/latest/install.html). Please refer to it for more information.

**Comment by 774911840** [comment]
Hi @Jacky1128 ,thanks for your answer,i also want know if this method could use in semantic segmentation?

**Comment by Lupin1998** [comment]
Hi, @774911840. Unfortunately, [AutoMix](https://arxiv.org/abs/2103.13027) only supports classification tasks because it requires a mixup classification loss (e.g., mixup with CE loss or BCE loss) to train the mixed sample generation process in the `MixBlock`. You can try to transfer it to downstream tasks like objection detection and semantic segmentation. For example, a simple way is first to adopt a multi-label classification loss with mixup to pre-train a `MixBlock` in [AutoMix](https://arxiv.org/abs/2103.13027) or [SAMix](https://arxiv.org/abs/2111.15454), and then generate augmented images (with instance-level objects) by the given labels to perform segmentation. Moreover, mixup augmentations in `OpenMixup` also support classification tasks only. To the best of my knowledge, [Mixup](https://arxiv.org/abs/1710.09412) and [CutMix](https://arxiv.org/abs/1905.04899) are the only two mixup techniques adopted in object detection and segmentation (e.g., in [Yolo.V4](https://arxiv.org/abs/2004.10934)).

**Comment by Lupin1998** [comment]
Thanks again for using our framework. I will close this issue if there is no more question. If you have more questions, you can reopen it or start a new issue.
